### DISCUSSION TITLE
IKEA lights turning on at brightness level 1 and randomly dimming themselves

### DISCUSSION THREAD

**Discussion by depau** [Koenkk/zigbee2mqtt#20783] [Devices]
I started having the problem described in the title a few months ago. The only unusual thing I can recall that happened on the day it started happening was a short black-out.

I own these lights:

- [LED1732G11](https://www.zigbee2mqtt.io/devices/LED1732G11.html#ikea-led1732g11) (TRADFRI bulb E27 WS opal 1000lm, firmware v2.3.091 and v2.3.095):
	- The issue is exactly as described in the title
	- The bulbs always turn on at level 1 when using a remote bound to OnOff+LevelCtrl+Scenes ([E1524/E1810](https://www.zigbee2mqtt.io/devices/E1524_E1810.html#ikea-e1524%252Fe1810))
	- I have some remotes with the older firmware that binds to groups and some with the newer firmware that binds to devices; the issue happens regardless of the binding method / firmware version
	- When turning them on from zigbee2mqtt they sometimes preserve the brightness level
- [LED1733G7](https://www.zigbee2mqtt.io/devices/LED1733G7.html#ikea-led1733g7) (TRADFRI bulb E14 WS opal 600lm, firmware v2.3.091)
	- It usually turns on at the last brightness level, but it dims itself to level 1 at least once a day
- [LED2003G10](https://www.zigbee2mqtt.io/devices/LED2003G10.html#ikea-led2003g10) (TRADFRI LED bulb E26/27 1100/1055/1160 lumen, dimmable, white spectrum, opal white, firmware v1.0.012):
	- Like the previous one but a lot less often (~weekly)

All the lights are 3 or 4 years old, except LED2003G10 which I bought a few months ago.

I tried factory resetting the lights and the remotes multiple times but it didn't help.

As a workaround I wrote [this script](https://github.com/depau/ikea-light-fixer) since I thought they were just defective (they're around 4 years old), and I immediately got [this message](https://github.com/depau/ikea-light-fixer/issues/1) from another person experiencing the same issue, who suggested it could be an issue with z2m.

Does anyone have any idea what could be going on?

**Comment by tjorim** [comment]
I have the same with an IKEA LED1545G12 (TRADFRI bulb E27 WS opal 980lm). The firmware version is 2.3.095 and build date 20220928, so I don't think I've updated recently. I cannot put a finger on when it exactly started but must indeed have been a few months.

**Reply by tjorim** [reply]
Apparently, there's already a (quite long) discussion about this in issue https://github.com/Koenkk/zigbee2mqtt/issues/19211.

**Reply by depau** [reply]
Good catch! I couldn't find it.

**Comment by depau** [comment]
This is indeed a duplicate of #19211 so I'll close this
